server/mlabns/util/resolver.py

```diff
@@ -186,18 +186,20 @@ def answer_query(self, query):
 
 
 class GeoResolverWithOptions(ResolverBase):
-    """Chooses the server geographically closest to the client."""
+    """Chooses the N geographically closest servers to the client."""
 
     def answer_query(self, query):
-        """Selects the geographically closest SliverTool.
+        """Selects the top N geographically closest SliverTools to the client.
 
         Args:
             query: A LookupQuery instance.
 
         Returns:
-            A SliverTool entity in case of success, or None if there is no
+            A list of SliverTool entities on success, or None if there is no
             SliverTool available that matches the query.
         """
+        # Return no more than MAX_RESULTS SliverTools in the result.
+        MAX_RESULTS = 4
```

Comment:
It's no longer user-definable as of this change. What do you think would be clearer in the description? I agree that it's not very clear, but I can't think of a better alternative. It feels strange to say "selects the top 4" and also strange to say "selects the top MAX_RESULTS." Do you have any suggestions?